README.md

```diff
@@ -94,8 +94,9 @@ Install [HomeBrew](https://docs.brew.sh/Installation) if not installed.
 
 ```sh
 rm -rf "$(brew --prefix)/share/ani-cli" "$(brew --prefix)/bin/ani-cli" && \
```

Comment:
unnecessary  `&& \newline`

Reply:
> Also two bugs I have noticed while testing:
> 
> * Menu isn't displayed until playback ends, even when playing one episode
> * History entries get duplicated (or rather one new instance is inserted) when resuming them. If you resume once again, it weirds out the script: it'll play the episode as many times as it is in the history, then finish with a similar screen to this:
>   ![image](https://user-images.githubusercontent.com/66648475/174159134-59282362-e22c-4ea9-9fb8-a60b983959ac.png)
>   I'll try to help to work these out, but because of the second I can't currently approve merging this - at least not as a major version.

okay... I will try to fix it tomorrow after exam..

Reply:
found the bug.. apparently it's [line 409](https://github.com/pystardust/ani-cli/pull/767/files#diff-194f14a8c6649316c12fa9d712dd46d7f2424da13786e02de699144517efc3a6R409)  that get slipped out while refactoring 
as i was testing something and it gave the same problem as u r mentioning..

Reply:
the line should be this..
```
grep -q "$selection_id" "$logfile" || printf "%s\t%s\n" "$selection_id" $((episode+1)) >> "$logfile"
```

Reply:
Implemented the line, it works now

Reply:
Also I have found the source of the other bug (it was my mistake when refactoring): the players were launched synchronously in all cases, where it should only be waited in the case of `auto_play`. Fixed.